kube/deploy.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euo pipefail # make bash quit if something weird happens
+
+export KUBE_NAMESPACE=${ENVIRONMENT}
```

Comment:
Looks like you're expecting `$ENVIRONMENT` to be one of `dev`, `qa`, `demo`, or `prod`. None of those are valid namespaces for DECS. If you want to deploy to both cs and wcs, it'd make sense to call the deployment step twice, in parallel.

Reply:
Good point - the initial project on gitlab was created for deployment to all namespaces. This project is only meant to create the image in quay.

Reply:
The relevant scripting was removed from both deploy.sh and drone.yml

Reply:
If we're not deploying I guess this `deploy.sh` in its entirely can go?